### PR TITLE
Created Method that spawns a Game Object

### PR DESCRIPTION
**Pull request by vreddi**
- Still a bunch of TODO left in the method
- Removed tweet_json.php from root directory. The file should be in the
  php directory

### CODE REVIEW

File: js/twitterApi.js [hunk 1]
```diff
@@ -60,13 +66,65 @@ $(document).ready(function(){
 
 			// So that no person is selected more than once
 			twitterUsersArr.splice(rand, 1);
-
-			console.log(twitterUsersArr.length);
 		}
```
Comment (sukritchhabra) [inline js/twitterApi.js]: I think we should get rid of all this unnecessary whitespace. Let me know if you're handling it, or if you want I'll merge it then take care of it.

Comment (sukritchhabra) [inline js/twitterApi.js:70]: I think we should get rid of all this unnecessary whitespace. Let me know if you're handling it, or if you want I'll merge it then take care of it.

File: js/twitterApi.js [hunk 2]
```diff
@@ -60,13 +66,65 @@ $(document).ready(function(){
...
+
+			// Fill in User Object Details
+			userObj['name'] = tweet['user']['name'];
+			userObj['handle'] = '@' + tweet['user']['screen_name'];
+			userObj['profilePicURL'] = tweet['user']['profile_img_url'];
+			userObj['followURL'] = "https://twitter.com/intent/follow?screen_name=" + userObj['handle'];
+
+			gameObj['correct'][tweet['user']['name']] = {};
+			gameObj['correct'][tweet['user']['name']]['userInfo'] = userObj;
+
```
Comment (sukritchhabra) [inline js/twitterApi.js:109]: unnecessary

File: js/twitterApi.js [hunk 3]
```diff
@@ -60,13 +66,65 @@ $(document).ready(function(){
...
+			tweetObj['tweetHTML'] = // TODO --> Add a function that takes in tweet and then create the HTML tweet text for return.
+			tweetObj['tweetText'] = tweet['text'];
+			tweetObj['numOfRetweets'] = tweet['retweet_count'];
+			tweetObj['numOfFavorites'] = tweet['favorite_count'];
+
+			// TODO --> Add tweet object to the Game Object
+
+		});
+
+		// TODO --> create shuffled incorrect object
```
Comment (sukritchhabra) [inline js/twitterApi.js:123]: TODO's should also be added to either comments in the pull reequest or an issue or both

File: js/twitterApi.js (hunk 1 above)
Comment (vreddi) [inline js/twitterApi.js:70]: I feel for JS files we can leave whitespace lines in between. it makes the code much more readable. Too many lines together is not easy on the eyes for navigation.

File: js/twitterApi.js (hunk 2 above)
Comment (vreddi) [inline js/twitterApi.js:109]: Why is that unnecessary? We need to the objects here right?

File: js/twitterApi.js (hunk 3 above)
Comment (vreddi) [inline js/twitterApi.js:123]: Note for next-time

**Pull request merged**